src/test/scala/org/broadinstitute/dsde/rawls/integrationtest/HttpGoogleServicesDAOSpec.scala

```diff
@@ -88,6 +90,31 @@ class HttpGoogleServicesDAOSpec extends FlatSpec with Matchers with IntegrationT
     intercept[GoogleJsonResponseException] { directory.groups.get(ownerGroup).execute() }
   }
 
+  it should "crud tokens" in {
+    val userInfo = UserInfo(null, null, 0, UUID.randomUUID().toString)
+    assertResult(None) { Await.result(gcsDAO.getToken(userInfo), Duration.Inf) }
+    assertResult(None) { Await.result(gcsDAO.getTokenDate(userInfo), Duration.Inf) }
+    val storeTime = DateTime.now()
+    Await.result(gcsDAO.storeToken(userInfo, "testtoken"), Duration.Inf)
+    assertResult(Some("testtoken")) { Await.result(gcsDAO.getToken(userInfo), Duration.Inf) }
+
+    // note that this assertion may fail if the runner's time is out of sync with google's
+    assert(!Await.result(gcsDAO.getTokenDate(userInfo), Duration.Inf).get.isBefore(storeTime))
```

Comment:
you could store twice and assert the new time is greater, this seems like it may be flaky